xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -970,7 +976,15 @@ bool CGUIWindowVideoBase::OnPlayMedia(int iItem, const std::string &player)
   if (m_thumbLoader.IsLoading())
     m_thumbLoader.StopAsync();
 
-  CServiceBroker::GetPlaylistPlayer().Play(itemCopy, player);
+  if (itemCopy->GetVideoInfoTag()->m_type == MediaTypeMovie ||
```

Comment:
Could you please add a `//! @todo get rid of special handling for movie versions`, so we can pick this up in a follow-up PR?